include/game/game.h

```diff
@@ -5,4 +5,10 @@
 
 void GameStart(void);
 
+#define GAME_SA1     1
+#define GAME_SA2     2
+#define GAME_SA3     3
+#define GAME_KATAM   4
```

Comment:
🤔 

I actually think these should be like `ENGINE` or `CORE_VERSION` and be part of the `core.h` header if we want to include KATAM here.

It makes sense to differentiate the core features from the game features IMO.

Reply:
I've been going back and forth on this myself.

Kinda don't wanna spam the codebase with `#define`s too much.
With your way we'd have something for both the core and game features, but usually you just wanna use these together, for matching, anyway.

Reply:
But you'd use `ENGINE_....` in the engine related code and `GAME_...` in the game related code. Atm referencing the game from the engine seems backwards. The engine is likely its own lib which just calls `GameStart`? I don't think there is anything in the engine code which is specific to sa2?